### ISSUE TITLE
map.NavigateTo Doesn't refresh the tiles

### ISSUE THREAD

**Issue by EduardsBrown** [pauldendulk/Mapsui#235]
Hi,

I am using map.NavigateTo to center on a point with a reasonable zoom level, that point moves on the map and when it gets too far the map tiles are not loaded. To load the tiles I need to move my Mouse on the map.

I am using:
Mapsui: 1.1.1
NETCore.UniversalWindowsPlatform: 6.0.1

Examples:
![image](https://user-images.githubusercontent.com/9280894/31776212-ce438a2e-b4e2-11e7-8229-5cc81e1e049e.png)
![image](https://user-images.githubusercontent.com/9280894/31776239-df88c498-b4e2-11e7-8c90-a4def1c6cfe6.png)

Is there a method that I can call to refresh the map? or is NavigateTo is not built for that? if so what should I use?

Thanks.

**Comment by pauldendulk** [comment]
Refresh should work properly with NavigateTo. I can not reproduce it in the Mapsui sample app.

Do you have the same problem if you use:
```MapControl.Map.NavigateTo(MapControl.Map.Viewport.Resolution /= 2);```
(this is easy for me to test with)

You use 'NETCore.UniversalWindowsPlatform: 6.0.1'. I am not familiar with this. Is this on option when you create a new project. Like this:
![image](https://user-images.githubusercontent.com/963462/31815209-4d06e514-b58c-11e7-96fa-9f67e5dc05ea.png)

**Comment by EduardsBrown** [comment]
Hi @pauldendulk,

I don't have the same issue when I use `MapControl.Map.NavigateTo(MapControl.Map.Viewport.Resolution /= 2);`

My UWP Min/Max version:
![image](https://user-images.githubusercontent.com/9280894/31816347-dd83d1ba-b587-11e7-9376-aabad3de3d54.png)
I need to use these versions as I have dependency on some .Net Standard 2.0 libraries.

![image](https://user-images.githubusercontent.com/9280894/31816376-03083e26-b588-11e7-93d0-45f8e26ec82c.png)

Thanks.

**Comment by pauldendulk** [comment]
ah, I can reproduce it. It is specific to UWP. Not sure yet what is happening.

**Comment by pauldendulk** [comment]
It is fixed in 1.1.2: https://www.nuget.org/packages/Mapsui/

Thanks for reporting!

**Comment by EduardsBrown** [comment]
I've tested it, and can confirm that this issue is fixed.

But when I updated to Mapsui 1.1.2, my references got messed up.
For example:
The compiler was expecting BruTile 0.22.1, but Mapsui has only pulled 0.21.1.
The compiler was expecting SkiaSharp & SkiaSharp.View 1.59.0, but Mapsui has only pulled 1.58.1.

It was saying that the packages version was different to the actual downloaded version.
I've fixed it by manually going through dependencies and lining them up.

I am not sure if that was my Visual Studios fault or your package.config mismatching.
